apps/site/src/scripts/component-demo-runtime.ts

```
@@ -1,3 +1,10 @@
+import type { HomepageDemoScenarioId } from '../components/homepage-demo/scenario-types';
+import {
+    createHomepageDemoHostCss,
+    createHomepageDemoInstanceSelector,
+    scopeHomepageDemoCss,
+} from '../components/homepage-demo/host-css';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)